backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/BpkDialog.kt

```diff
@@ -251,10 +261,17 @@ fun BpkImageDialog(
             linkButton?.let { Dialog.Button(BpkButtonType.Link, linkButton) },
         ),
         onDismissRequest = onDismissRequest,
-        properties = properties,
+        properties = getDialogProperties(dismissOnBackPress, dismissOnClickOutside),
         textAlign = textAlign,
         content = content,
     )
 }
 
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
I don't think `ExperimentalComposeUiApi` is needed.

Reply:
![Screenshot 2023-04-26 at 12 07 13](https://user-images.githubusercontent.com/92924979/234557266-52053c51-29ca-4eff-8287-11fc8672f956.png)

it seems to be, I get this error otherwise.

Reply:
I removed it and it works. Probably a gradle sync fixes it